api/app/fireweather_bot/common.py

```diff
@@ -81,7 +82,9 @@ def get_station_names_to_codes() -> dict:
     """ Helper function to create dictionary of (station_name: station_code) key-value pairs
     Is used when replacing station names with station IDs in dataframe
     """
-    station_data = _get_stations_local()
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
+    station_data = loop.run_until_complete(get_stations())
```

Comment:
Can you explain why it has to be done this way?

Reply:
The API runs asynchronously, and when we make http requests from inside the API, we make them asynchronously as well. When we run the bot, it's not running asynchronously - and you can't just call async code from non async code. So that bit is just putting in a block on get_stations. If there's a simpler way to do it, I'd love to find out.
We used to call synchronous code from the bot, but I've re-factored it to not care if it's going against wfx (async) or local db (sync) - so I've had to introduce this bit.